application/src/main/java/uk/nhs/digital/uec/api/config/SecurityConfig.java

```diff
@@ -42,7 +43,6 @@ protected void configure(HttpSecurity http) throws Exception {
         Arrays.asList(
             HEALTH_CHECK_READINESS_URL,
             HEALTH_CHECK_LIVENESS_URL,
-            WELCOME_URL,
```

Comment:
All the endpoints in this list shouldn't need authentication, so you should already have been able to get to the welcome page without needing to make changes